xdx.py

```diff
@@ -5,16 +5,17 @@
 from random import randint as randy
 
 def roll_baby_roll(left, size):
-    if left < 1:
-        return 0
-    elif left == 1:
-        return randy(1, size)
-    else:
-        return roll_baby_roll(left-1, size) + randy(1, size)
+	for i in range(left):
+		yield randy(1,size)
+
 
 def main():
     dice = sys.argv[1].split('d')
-    print(roll_baby_roll(int(dice[0]), int(dice[1])))
+    tot = 0
+    for dice in roll_baby_roll(int(dice[0]), int(dice[1])):
+        print(dice)
```

Comment:
Can you add a roll num here so there aren't too many numbers being returned one after the other?

Reply:
No. Git good 😎

Reply:
EX: Roll 1: 
Roll 2:
Roll 3:
...

Reply:
[Happy](https://github.com/masonseeger/xdx-superiority-project/pull/1/commits/181890ddbf7c893c1c2e368affe72c78887eea76)?